src/path_converters.h

```diff
@@ -677,8 +703,9 @@ class PathSimplifier : protected EmbeddedQueue<9>
                 m_origdNorm2 = m_origdx * m_origdx + m_origdy * m_origdy;
 
                 // set all the variables to reflect this new orig vector
-                m_dnorm2Max = m_origdNorm2;
-                m_lastMax = true;
+                m_dnorm2ForwardMax = m_origdNorm2;
+                m_dnorm2BackwardMax = 0.0;
+                m_lastForwardMax = true;
```

Comment:
I'm pretty sure it can only be false at this point in the code. It wouldn't hurt though, so if you're worried about it I'm OK adding that in.